src/style.css

```diff
@@ -151,4 +152,431 @@ nav.active a img {
   nav ul li a:focus img {
     transform: scale(1.1);
   }
+
+  /* desktop Menu update */
+  .body_container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    margin-left: 0;
+  }
+
+  .project_menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 80%;
+  }
+
+  .menu_list {
+    display: flex;
+    align-items: center;
+  }
+
+  .menu_element {
+    font-size: 20px;
+    font-weight: bold;
+  }
+
+  .navbar {
+    width: 60%;
+    justify-content: center;
+  }
+
```

Comment:
I've noticed that there are duplicate styles within the code. It's important to maintain consistency and efficiency in your styling, so please review and eliminate any redundant or duplicate styles to keep the codebase clean and organized.